src/processor/call_handler.rs

```diff
@@ -0,0 +1,130 @@
+use crate::args::CallHandlerArgs;
+use crate::consts::EXTERNAL_CALL_HANDLER_DISCRIMINATOR;
+use crate::ephemeral_balance_seeds_from_payer;
+use crate::processor::utils::loaders::{
+    load_initialized_validator_fees_vault, load_owned_pda, load_pda, load_signer,
+};
+
+use borsh::BorshDeserialize;
+use solana_program::account_info::next_account_info;
+use solana_program::account_info::AccountInfo;
+use solana_program::entrypoint::ProgramResult;
+use solana_program::instruction::{AccountMeta, Instruction};
+use solana_program::program::invoke_signed;
+use solana_program::program_error::ProgramError;
+use solana_program::pubkey::Pubkey;
+use solana_program::{msg, system_program};
+
+pub const INVALID_ESCROW_PDA: &str = "invalid escrow pda in CallHandler";
+pub const INVALID_ESCROW_OWNER: &str = "escrow can not be delegated in CallHandler";
+pub const ACTION_FAILED_MSG: &str = "user action failed";
+
+/// Delegates an account
```

Comment:
Copy/paste typo: change to describe call handler

Reply:
Changed